backend/src/api/app.ts

```diff
@@ -305,6 +305,10 @@ authenticatedRoute.get(
 );
 
 authenticatedRoute.get('/organizations', handlerToExpress(organizations.list));
+authenticatedRoute.get(
+  '/organizations/all',
```

Comment:
Why do we need this new endpoint? Could we just use `/organizations` instead and maybe pass a parameter?

Reply:
This is an excellent idea.  After some research and searching in our code, I couldn't find a good way to send a parameter or body along with a get api call without changing the api calls as well.   There also isn't really a route that does anything similar that we could expand.  

I am open to and appreciate any feedback. :) If there is a way to do this, I would love to learn.

Reply:
(If we do need to access parameters in a GET API call, we can access the query string using `req.query` -- also see https://www.tutorialspoint.com/getting-query-string-variables-in-express-js).

But, looking at the code again, I don't think we need to pass a parameter. Can't we just call `/organizations` and it will do the same thing? So we can replace `/organizations/all` with `/organizations` instead?

Reply:
Ok, sounds good! Let me know if you'd like to chat more.

Reply:
> So we can replace `/organizations/all` with `/organizations` instead?

So do you think this is possible, or would that not be?

Reply:
@epicfaace, apologies, I was unclear in my comment, then went out of town for a long weekend. :) Yes I believe this is possible with the page you linked.   I will just have it pass in an ?all=true for the one that needs all of them and we can extend the functionality of the organizations route.